src/components/Minecraft/Create/GameTypeSelectForm.tsx

```diff
@@ -0,0 +1,58 @@
+import { RadioGroup } from '@headlessui/react';
+import { HandlerGameType } from 'bindings/HandlerGameType';
+import { GameType } from 'bindings/InstanceInfo';
+import clsx from 'clsx';
+import Spinner from 'components/DashboardLayout/Spinner';
+import { InstanceGameTypes } from 'data/InstanceGameTypes';
+import SelectGameCard from './SelectGameCard';
+import { gameTypeInfoFromHandlerType } from 'data/GameTypeMappings';
+export default function GameTypeSelectForm({
+  gameType,
+  setGameType,
+  className,
+}: {
+  gameType: HandlerGameType;
+  setGameType: (gameType: HandlerGameType) => void;
+  className?: string;
+}) {
+  const { data: game_types, isLoading, error } = InstanceGameTypes();
+  if (!game_types || isLoading) {
+    return <Spinner />;
+  }
+
+  return (
+    <div className={className}>
+      <p className="text-left text-h2 font-bold tracking-medium text-gray-300">
+        Select Game
+      </p>
+      <p className="text-left text-medium font-bold italic leading-5 tracking-medium text-white/50">
+        What will your server be used for?
+      </p>
+      <div className="mt-9 box-border grid h-full grid-cols-2 gap-9">
+        {game_types.map((game) => {
+          const { title, description, game_type } =
+            gameTypeInfoFromHandlerType[game];
+          return (
+            <div key={game}>
+              <SelectGameCard
+                title={title}
+                description={description}
+                game_type={game_type}
+                className={clsx(
```

Comment:
The hover style looks a little weird on selected elements, is this intended?
Also, why are we hard coding the styles here, instead of inside the `SelectGameCard`
![image](https://user-images.githubusercontent.com/44907675/222987568-77609727-d1c0-48eb-a7f4-3f82df4ef85d.png)

Reply:
@Arcslogger Thoughts? This seems to also apply to other buttons like the icon buttons and user menu